schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals(
   date_of_birth date,
   escape_attemps INT NOT NULL,
   neutered boolean,
-  weight_kg decimal(5, 2)
+  weight_kg decimal(5, 2),
+  species varchar(50)
```

Comment:
- [ ] Kindly note that regarding the requirement you should not directly add the `species` column in the animal table instead you should add it by altering the table. hint  :point_down: 

```sql
   ALTER TABLE animals ADD species varchar(255);
```

Reply:
Thank You for the review